src/styles.scss

```diff
@@ -33,7 +33,7 @@ app-root {
   flex-direction: column;
 }
 .router-container {
-  padding-bottom: 140px; //remove footer height
+  padding-bottom: 240px; //remove footer height
```

Comment:
Is this ok? I am afraid my break  the home page and other pages

Reply:
Yes, actually it is a rollback